### PR TITLE
Unit test: for catch part in JSONObject

### PR DESCRIPTION
**Pull request by UvrajSB**
## Purpose / Description
Unit test: for catch part of the getBoolean, getInt, getLong, getStrong functions in JSONObject

## How Has This Been Tested?
I have tested by running it on my local machine
image

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [ ] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [ ] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/test/java/com/ichi2/utils/JSONObjectTest.kt [hunk 1]
```diff
@@ -1084,6 +1084,23 @@ class JSONObjectTest {
         }
     }
 
+    @Test
+    fun testGetThrows() {
+        val `object` = JSONObject()
+        Assert.assertThrows(JSONException::class.java) {
```
Comment (david-allison) [inline AnkiDroid/src/test/java/com/ichi2/utils/JSONObjectTest.kt:1090]: Inside `AnkiAssert`:

`assertThrows<JSONException> { }`

**Review by david-allison — CHANGES_REQUESTED** [review]
minor request to make the code slightly more readable

**Review by mikehardy — COMMENTED** [review]
This looks okay to me with the minor thought that I'm not sure if the name `object` is poorly chosen since it seems to require backticks to differentiate from a reserved word. Couldn't any non-reserved word be used there to avoid the backticks?

**Comment by mikehardy** [comment]
> This change is not just there for my part but for the all the other tests as well and it has already been acknowledged as a kotlin cleanup in the file. So it doesn't come under the scope of this PR in my opinion.
> I'll change that and open a new PR for that

Oh - good point - I'm +1 then - on the new PR please use a descriptive title, I see it linked here as "Kotlin Cleanup" which as a reviewer makes me think "wow, they cleaned up ALL the KotlinCleanup fantastic - but it's going to be a big PR" ;-). Probably a file name would be good in title

**Review by mikehardy — APPROVED** [review]

**Comment by mikehardy** [comment]
also it will probably conflict when I merge this - all the details all the time - that's programming :sweat_smile:

File: AnkiDroid/src/test/java/com/ichi2/utils/JSONObjectTest.kt (hunk 1 above)
Comment (mikehardy) [reply AnkiDroid/src/test/java/com/ichi2/utils/JSONObjectTest.kt]: this has been addressed I checked

**Comment by mikehardy** [comment]
Sorry for the extra noise via comments @UvrajSB - I see that you were actually just commenting on the general Kotlin Cleanup issue now, so of course the comment makes sense. Ignore me - will merge this in a moment, running re-integration tests against head of main branch now locally

**Pull request merged**